### ISSUE TITLE
onnx to tenorRT engine: [TensorRT] ERROR: ../builder/cudnnBuilderGraphOptimizer.cpp (1470) - Assertion Error in fusePattern: 0 (eltwiseSuccessorsIdx.size() == eltwiseSuccessors.size())

### ISSUE THREAD

**Issue by Pudge-tao** [NVIDIA/TensorRT#891]
## Environment

**TensorRT Version**: 7.1.3.4
**GPU Type**: Tesla t4
**Nvidia Driver Version**: 
**CUDA Version**: 10.2
**CUDNN Version**: 8.0
**Operating System + Version**: centos7
**Python Version (if applicable)**: 3.7
**TensorFlow Version (if applicable)**: 2.2
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if container which image + tag)**: 

## Relevant Files

layers include "tf.nn.l2_normalize"

**Comment by Pudge-tao** [comment]
when i remove “tf.nn.l2_normalize”, there is no error.  how to l2 normalize in tensorRT. i want to normalize output in GPU

**Comment by mk-nvidia** [comment]
@Pudge-tao please attach an onnx model that reproduces the problem. Without that it's very hard to determine what's going wrong. Also consider trying TensorRT 7.2 which has several bug fixes.

**Comment by ttyio** [comment]
Closing since no activity in this thread for more than 3 weeks, please reopen if you still have question, thanks!
